content/qcto/data-science-practitioner/km-01/km-01-kt05/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+_db_id:
+content_type: topic
+ready: true
+title: Applications of Data Science
+---
+
+# Applications of Data Science
```

Comment:
I suggest that you follow the [guidelines here](https://github.com/Umuzi-org/ACN-syllabus/blob/develop/contribute.md#-a-note-on-headings)(https://github.com/Umuzi-org/ACN-syllabus/blob/develop/contribute.md#-a-note-on-headings) on heading hierarchy.